pkg/fakes/mock_blockchain.go

```diff
@@ -47,6 +47,7 @@ type MockBlockChain struct {
 	logQueryReturnLogs                 []types.Log
 	node                               core.Node
 	storageValuesToReturn              map[common.Address]map[int64][]byte
+	GetHeadersByNumbersErr             error
```

Comment:
Should this go up with the other public fields? That kinda looks like the convention, but if it isn't and that's a coincidence feel free to tell me to stop nitpicking.